valgrind_suppressions.txt

```diff
@@ -14,4 +14,20 @@
    fun:dlopen@@GLIBC_*
    fun:*QLibraryPrivate*load_sysEv
    fun:*QLibraryPrivate*loadEv
+}
+{
+   qt_quick_test_setup_with_platform_offscreen
+   Memcheck:Leak
+   match-leak-kinds: definite
+   fun:_Znwm
+   obj:/usr/lib/x86_64-linux-gnu/qt5/plugins/platforms/libqoffscreen.so
```

Comment:
wonder if we should remove the arch from the path here, so it can work on other platforms ? eg just have `*/plugins/platforms/libqoffscreen.so` or something

Reply:
Done lets see if that still passes